src/marqo/tensor_search/validation.py

```diff
@@ -18,6 +18,7 @@
 from marqo.tensor_search.models.delete_docs_objects import MqDeleteDocsRequest
 from marqo.tensor_search.models.settings_object import settings_schema
 from marqo.tensor_search.models.mappings_object import mappings_schema, multimodal_combination_schema
+import numpy as np
```

Comment:
it doesn't look like numpy is used anywhere in this file and therefore is a unnecessary import

Reply:
fixed. also removed other unnecessary imports.